### ISSUE TITLE
Docker error with v2.31.0 : s6-supervise filebrowser: warning: unable to spawn ./run (waiting 60 seconds): Permission denied

### ISSUE THREAD

**Issue by romainsi** [filebrowser/filebrowser#3442]
**Description**
With last update (v2.31.0) of FileBrowser on Docker (filebrowser-s6) container failed to start with error :
`s6-supervise filebrowser: warning: unable to spawn ./run (waiting 60 seconds): Permission denied`

Example of my docker-compose.yml :
```
services:
  filebrowser:
    image: filebrowser/filebrowser:s6
    container_name: filebrowser
    restart: always
    volumes:
      - /home/example:/srv
      - /opt/filebrowser/filebrowser.db:/database/filebrowser.db
      - /opt/filebrowser/settings.json:/config/settings.json
      - /opt/filebrowser/filebrowser.log:/config/filebrowser.log
      - /opt/filebrowser/custom:/config/custom
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Paris
    ports:
      - 127.0.0.1:8888:80 # HTTP port
```

This configuration working before update.

So I switched back to an old version  :
`image: filebrowser/filebrowser:v2.30.0-s6`

But settings.json does not seem to have the correct rights after installation in the container.
I solved the problem temporarily by opening a bash in the container and restoring rights with the command: 
`chown abc:users /config/settings.json`.

I use Docker version 27.2.0 and AlmaLinux release 9.4.

Thanks in advance.

**Comment by masterwishx** [comment]
have same issue

**Comment by milindpatel63** [comment]
Same issue here too

**Comment by masterwishx** [comment]
@o1egl can you please check it?

**Comment by mpissarello** [comment]
Same issue here.
Attached [here](https://github.com/user-attachments/files/16818702/_local-filebrowser_logs.txt) the starting log: it seems a problem of the LinuxServer baseimage that has been deprecated...

**Comment by lamousis** [comment]
Also same issue.
I created manually file browser.db as mentioned in the documentation but still error.
I tried it with portainer and state is unhealthy

**Comment by 0xN1nja** [comment]
Same

**Comment by masterwishx** [comment]
> Same issue here. Attached [here](https://github.com/user-attachments/files/16818702/_local-filebrowser_logs.txt) the starting log: it seems a problem of the LinuxServer baseimage that has been deprecated...

Also have : 

![image](https://github.com/user-attachments/assets/c7e3a755-0c6c-4629-ab39-91232a0bc012)

**Comment by ryanwinter** [comment]
This issue isn't with the deprecated image, its lack of execute permissions on the two etc scripts:

- https://github.com/filebrowser/filebrowser/blob/master/docker/root/etc/cont-init.d/20-config
- https://github.com/filebrowser/filebrowser/blob/master/docker/root/etc/services.d/filebrowser/run

The first one fails which results in some possible bad permissions, and the failure to install the default config file.
The second actually starts the service.

Not sure how this worked in the past.

**Comment by ryanwinter** [comment]
Temporary workaround is to add execute permissions via a console and restart. Pulling a new container will revert back to broken state though.

```
chmod a+x /etc/cont-init.d/20-config
chmod a+x /etc/services.d/filebrowser/run

```

**Comment by masterwishx** [comment]
> Temporary workaround is to add execute permissions via a console and restart. Pulling a new container will revert back to broken state though.
> 
> ```
> chmod a+x /etc/cont-init.d/20-config
> chmod a+x /etc/services.d/filebrowser/run
> ```

Thanks but already moved to alpine version that working fine

**Comment by ryanwinter** [comment]
> > Temporary workaround is to add execute permissions via a console and restart. Pulling a new container will revert back to broken state though.
> > ```
> > chmod a+x /etc/cont-init.d/20-config
> > chmod a+x /etc/services.d/filebrowser/run
> > ```
> 
> Thanks but already moved to alpine version that working fine

Did the alpine 3.20 version fix the issue?

**Comment by ryanwinter** [comment]
Oh yeah weird, now that latest s6 in alpine 3.20 doesnt care about the execute permissions. I guess this was a regression in a new version of alpine 3.17.

**Comment by masterwishx** [comment]
> > > Temporary workaround is to add execute permissions via a console and restart. Pulling a new container will revert back to broken state though.
> > > ```
> > > chmod a+x /etc/cont-init.d/20-config
> > > chmod a+x /etc/services.d/filebrowser/run
> > > ```
> > 
> > 
> > Thanks but already moved to alpine version that working fine
> 
> Did the alpine 3.20 version fix the issue?

i meaned `latest` version of filebrowser

**Comment by o1egl** [comment]
Fixed in v2.31.1
